share/completions/git.fish

```diff
@@ -1870,3 +1870,18 @@ complete -c git -n '__fish_git_using_command bisect; and __fish_seen_argument --
 
 ## Custom commands (git-* commands installed in the PATH)
 complete -c git -n __fish_git_needs_command -a '(__fish_git_custom_commands)' -d 'Custom command'
+# source git-* commands' autocompletion file if exists
+set -l __fish_git_custom_commands_completion
+for complete_path in $fish_complete_path
+    for git_ext in $complete_path/git-*.fish
```

Comment:
This can actually be done in a single loop:

```fish
for git_ext in $fish_complete_path/git-*.fish
```

Reply:
Thanks for the suggestion! I didn't know you can do that in shell script